src/action-parser.ts

```diff
@@ -117,8 +117,7 @@ function parseLocation(location: string, takingFaction: RootFaction): RootLocati
     return +location;
   }
 
-  console.error(`Could not parse location: "${location}"`);
-  return null as RootLocation;
+  throw `Could not parse location: "${location}"`;
```

Comment:
Can you do `throw new Error(msg)` instead? Throwing strings, while possible, isn't trackable.

Reply:
Yes, done! 👍